geoNet/process_observed.py

```diff
@@ -0,0 +1,370 @@
+"""Processes V1A or miniseed files from observed events and saves these as text files"""
```

Comment:
Probably worth stating that you are leaving the units as is.

Reply:
Resulting units are always in g?

Reply:
For V1A the interface returns them as in units g, and for miniseed I divide by g

Reply:
Oh sweet. Are you able to add that to the comment?

Reply:
Done